drafts/draft-20005.adoc

```diff
@@ -0,0 +1,223 @@
+----
+
+To configure the Liberty logs to output logs in the new simple console format, you just have to set the following logging server configuration, in the bootstrap.properties, server.xml, or in the server.env (environment variables).
+
+==== bootstrap.properties
+`com.ibm.ws.logging.console.format=simple`
+
+==== server.xml
+`<logging consoleFormat="simple"/>`
+
+==== server.env
+`WLP_LOGGING_CONSOLE_FORMAT=simple`
+
+
+[#JLO]
+=== Omit specified fields from JSON logging output
+
+In Liberty, users have the option to format their server logs in basic or JSON format. When logs are in JSON format, users have to specify the sources (message, trace, accessLog, ffdc, audit) they want to send to messages.log or console.log/standard-out.
+
+Previously, Liberty has always had the default fields in the JSON output. Users can now specify the JSON fields they want to omit. This feature adds an option for users to omit JSON fields in the JSON logging process. The option to omit JSON field names in Liberty is extremely useful, as users may not want certain default fields provided by Liberty in their JSON output. Undesired fields add to the size of the records which wastes network I/O during record transmissions and waste space in downstream log aggregation tools. Now users have the option to only emit the fields they need so they can send to downstream log aggregation tools without using more space and I/O than necessary.
+For someone running liberty in docker containers, with a single server in each container, they may not want to include the JSON fields that represent the server name and user directory, for example.
+
+When logs are in JSON format, you can use the existing `jsonFieldMappings` attribute to omit fields. The attribute initially was only used for renaming field names. For renaming JSON field name, the format is specified as `source:defaultFieldName:newFieldName` or `defaultFieldName:newFieldName`. To omit `defaultFieldName`, leave `newFieldName` empty. For example, to omit a field for all sources, use the following format: `defaultFieldName:` . To omit a field for a specific source, use the following format: `source:defaultFieldName:` where source is the source you want to specify (such as message, trace, accessLog, ffdc, audit).
+
+An example of omitting JSON fields by adding the following to bootstrap.properties:
+`com.ibm.ws.logging.json.field.mappings=trace:ibm_userDir: ,ibm_datetime:`.
+
+You can find more information by viewing link:https://www.ibm.com/support/knowledgecenter/en/SSEQTP_liberty/com.ibm.websphere.wlp.doc/ae/rwlp_logging.html[Logging and Trace] on IBM Knowledge Center or on the link:https://www.ibm.com/support/knowledgecenter/en/SSEQTP_liberty/com.ibm.websphere.wlp.doc/ae/rwlp_logging.html[Open Liberty Logging docs]
+
+[#EPT]
+=== Persistent EJB Timers coordination and failover across members
```

Comment:
We should lead with this since it is the most significant new function in the post.